### PR TITLE
Back to Code button appears in all the files

### PR DESCRIPTION
**Pull request by abchatra**
Simple fix to address https://github.com/Microsoft/pxt/issues/711

### CODE REVIEW

File: webapp/src/monaco.tsx [hunk 1]
```diff
@@ -143,10 +143,15 @@ export class Editor extends srceditor.Editor {
...
     menu(): JSX.Element {
-        if (!this.hasBlocks()) return null
-        return <sui.Item class="blocks-menuitem" textClass="landscape only" text={lf("Blocks") } icon="puzzle" onClick={() => this.openBlocks() }
-                tooltip={lf("Convert code to Blocks")} tooltipPosition="bottom left"
-         />
+        let editor = pkg.mainEditorPkg(); 
+        if (this.currFile != editor.files["main.ts"]) {
+            return (<sui.Item text={lf("Back to Code") } icon={"align left"} onClick={() => this.parent.setFile(editor.files["main.ts"]) } />);
+        }
+        else if (editor.files["main.blocks"]) { //if main.blocks file present
```
Comment (pelikhan) [inline webapp/src/monaco.tsx]: You should go back to blocks by default.

File: webapp/src/monaco.tsx [hunk 2]
```diff
@@ -143,10 +143,15 @@ export class Editor extends srceditor.Editor {
...
+            return ( <sui.Item class="blocks-menuitem" textClass="landscape only" text={lf("Blocks") } icon="puzzle" onClick={() => this.openBlocks() }
```
Comment (pelikhan) [inline webapp/src/monaco.tsx]: Why not call it "Back to Code"?

Comment (abchatra) [inline webapp/src/monaco.tsx]: Not sure I get this comment. It is called "Back to Code". Else part deals with case where you are in typescript view and now want to go back to Blocks, if there is a main.blocks file.

File: webapp/src/monaco.tsx (hunk 1 above)
Comment (pelikhan) [inline webapp/src/monaco.tsx]: Good point!

File: webapp/src/monaco.tsx (hunk 2 above)
Comment (pelikhan) [inline webapp/src/monaco.tsx]: Looks good

**Pull request merged**